apps/remote_control/lib/lexical/remote_control/search/store.ex

```diff
@@ -34,12 +37,8 @@ defmodule Lexical.RemoteControl.Search.Store do
     GenServer.stop(__MODULE__)
   end
 
-  def metadata do
-    GenServer.call(__MODULE__, :metadata)
-  end
-
   def all do
-    GenServer.call(__MODULE__, :all)
+    GenServer.call(__MODULE__, :all, 20_000)
```

Comment:
I think this timeout is here to match the default 20s for indexing, but that may not be necessary with #371.

Reply:
correct. Removing.